invenio_accounts/testutils.py

```diff
@@ -63,11 +66,25 @@ def create_test_user(email='test@test.org',
     return user
 
 
-def login_user_via_view(client, email, password, login_url=None):
-    """Attempt to log the given user in via the 'login' view on the client.
+def login_user_via_view(client, email=None, password=None, user=None,
```

Comment:
I'm sorry but I'm not a big fan of adding `user` to the API. Can we instead create a helper function in tests that sets email and password?